### PR TITLE
Longhorn 1.2.3 & 1.1.3 for v2.6

### PR DESCRIPTION
**Pull request by innobead**
Signed-off-by: David Ko <dko@suse.com>

cc @khushboo-rancher @meldafrawi @kaxing

### CODE REVIEW

**Comment by khushboo-rancher** [comment]
@chanow816 has validated the chart v1.1.3 on rancher v2.6-head - LGTM

Validation of chart v1.2.3 is pending.

File: charts/longhorn-1.1/longhorn/100.0.1+up1.1.3-rc2/Chart.yaml
Comment (jiaqiluo) [inline charts/longhorn-1.1/longhorn/100.0.1+up1.1.3-rc2/Chart.yaml:8]: can we also add the following 2 annotations:
```
catalog.cattle.io/kube-version:  <the same value as the kubeVersion in chart.yaml> 
catalog.cattle.io/upstream-version: <longhorn's version>
```

**Review by jiaqiluo — CHANGES_REQUESTED** [review]

**Review by aiyengar2 — CHANGES_REQUESTED** [review]
I haven't checked this one in depth yet, but I assume it will be the same concerns as https://github.com/rancher/charts/pull/1639#pullrequestreview-828184086. Can you address those Chart.yaml changes here as well?

I can do a deeper review of this once those changes have been addressed to see if there are any more concerns.

File: charts/longhorn-1.1/longhorn/100.0.1+up1.1.3-rc2/Chart.yaml
Comment (innobead) [reply charts/longhorn-1.1/longhorn/100.0.1+up1.1.3-rc2/Chart.yaml]: Thanks for the info about this new change. Added.

**Comment by innobead** [comment]
> I haven't checked this one in depth yet, but I assume it will be the same concerns as [#1639 (review)](https://github.com/rancher/charts/pull/1639#pullrequestreview-828184086). Can you address those Chart.yaml changes here as well?
> 
> I can do a deeper review of this once those changes have been addressed to see if there are any more concerns.

Fixed.

**Review by jiaqiluo — APPROVED** [review]
LGTM

File: packages/longhorn-1.1/templates/crd-template/Chart.yaml
Comment (aiyengar2) [inline packages/longhorn-1.1/templates/crd-template/Chart.yaml:2]: Same question as https://github.com/rancher/charts/pull/1639#discussion_r768911771. Can merge if this is okay.

**Review by aiyengar2 — APPROVED** [review]
Same concern as in https://github.com/rancher/charts/pull/1639#discussion_r768911771, but approving since the PR is good to go as long as that question does not spawn any more changes.

**Comment by innobead** [comment]
We should not merge this until QA verify. We need to update again when GA. cc @cbron

**Pull request merged**
